### DISCUSSION TITLE
Possibility of getting the bundled pandoc version increased in v1.4 pre-release?

### DISCUSSION THREAD

**Discussion by coatless** [quarto-dev/quarto-cli#7048] [Q&A] [answered]
### Description

There's been a gnarly issue preventing the inclusion of `RawBlocks` inside the `revealjs` format. (c.f.  https://github.com/quarto-dev/quarto-cli/issues/5666, upstream: https://github.com/jgm/pandoc/issues/8880, extension: https://github.com/coatless/quarto-webr/issues/14). The fix here requires an upstream change inside of Pandoc. 

If I dedicate some cycles to handle the upstream change, would it be possible to get the included pandoc version in pre-release v1.4 updated to the version containing the fix? Or would the target for increasing the included pandoc version be v1.5?

**Comment by cscheid** [comment, marked as answer]
We update Pandoc much more aggressively than other dependencies, and we'll almost certainly update it again before 1.4 is out.

**Reply by cscheid** [reply]
(Just as an example, I'd love to ship 1.4 with this merged and our workaround removed: https://github.com/jgm/pandoc/issues/9104)

**Reply by coatless** [reply]
Awesome! Thank you for me know. I'll start pushing on the upstream fix.

**Reply by mcanouil** [reply]
If I am not mistaken the current version of Pandoc embedded in Quarto 1.4 is the latest release to date (3.1.8)

**Reply by cscheid** [reply]
@mcanouil Yes, but @coatless is asking about the possibility of upgrading it again (which is not how we handle other dependencies). And Pandoc is special in that sense.
